api.yaml

```diff
@@ -133,8 +170,26 @@ endpoints:
             integration.request.querystring.limit: method.request.querystring.limit
             integration.request.querystring.offset: method.request.querystring.offset
             integration.request.querystring.page: method.request.querystring.page
+            integration.request.querystring.page_size: method.request.querystring.page_size
           responses:
             200:
+              responseTemplates:
+                application/json: |
+                    #set($inputRoot = $input.path('$')) #set($host = $stageVariables.gateway_host)
+                    #macro (updateURL $url)
```

Comment:
I wonder if there is a way to not repeat this logic in so many different places?